src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/OperationFormatter.cs

```diff
@@ -513,15 +513,15 @@ internal static void GetActions(OperationDescription description, XmlDictionary
             }
         }
 
-        internal static NetDispatcherFaultException CreateDeserializationFailedFault(string reason, Exception innerException)
+        public static NetDispatcherFaultException CreateDeserializationFailedFault(string reason, Exception innerException)
         {
             reason = SR.Format(SR.SFxDeserializationFailed1, reason);
             FaultCode code = new FaultCode(FaultCodeConstants.Codes.DeserializationFailed, FaultCodeConstants.Namespaces.NetDispatch);
             code = FaultCode.CreateSenderFaultCode(code);
             return new NetDispatcherFaultException(reason, code, innerException);
         }
 
-        internal static void TraceAndSkipElement(XmlReader xmlReader)
+        public static void TraceAndSkipElement(XmlReader xmlReader)
         {
```

Comment:
This is a static method so can be duplicated on DataContractJsonSerializerOperationFormatter. Keep it internal here.

Reply:
This has been addressed in b068469e08af294d6db6a50cb5b3ea9266f26c39.